rclrs/src/rcl_bindings.rs

```diff
@@ -1,13 +1,7 @@
-#[allow(dead_code)]
+#![allow(dead_code)]
```

Comment:
Since you're changing this file, could you rename it to something other than `rcl_bindings.rs`? Or rename the `bindgen` file. Either way, it's sometimes a bit confusing when referring to `rcl_bindings.rs`, which is of the two.

Reply:
Sure, I renamed the bindgen one to `rcl_bindings_generated.rs`.